### PR TITLE
WIP ICU-12968 remove icudata.jar from icu4j tree

### PR DESCRIPTION
**Pull request by srl295**
- improve build of data
- :construction: does NOT work on windows
- add a convenience script on the icu4j side
- probably some other holes here

##### Checklist

- [x] Issue filed at https://unicode-org.atlassian.net/browse/ICU-12968
- [x] Update PR title to include Issue number
- [ ] Issue accepted (kind of)
- [ ] Documentation is changed or added (kind of)

### CODE REVIEW

**Comment by srl295** [comment]
https://travis-ci.org/unicode-org/icu/jobs/431690519#L5651 successfully built ICU4J with no .jar files in the repository… (added 4 minutes to build time)

**Comment by srl295** [comment]
https://github.com/unicode-org/icu/commit/b8c4e91dc7e7130bd9282bbd178bc0ed6af41fdf#diff-d32048782dc30c3615e42837d446d6b4 is an alternate approach ( and eerily similar, written a year and a half earlier which I obviously forgot about ) where icu4j calls icu4c to build.

**Comment by markusicu** [comment]
@srl295 are you intending to push this further?

File: icu4j/readme.html [hunk 1]
```diff
@@ -525,6 +525,15 @@ <h2 class="doc"><a name="HowToInstallJavac"></a>How to Install and Build</h2>
     </pre>
   </li>
 </ul>
+<p>
+If you have downloaded ICU4J from the git source repository, an additional step is needed:
+building ICU4C and its data. On POSIX systems, you can run the shell script to automatically
+build and install ICU data:
```
Comment (srl295) [inline icu4j/readme.html:531]: "on windows systems install WSL and try again"

**Pull request closed without merging**